File: myExpenses/src/main/java/org/totschnig/myexpenses/activity/HistoryActivity.java
package org.totschnig.myexpenses.activity;

import android.os.Bundle;

import org.totschnig.myexpenses.R;

public class HistoryActivity extends ProtectedFragmentActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.history);
    setupToolbar(true);
  }
}

File: myExpenses/src/main/java/org/totschnig/myexpenses/activity/HistoryActivity.kt
package org.totschnig.myexpenses.activity

import android.os.Bundle
import org.totschnig.myexpenses.fragment.HistoryChart

class HistoryActivity : ProtectedFragmentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setupWithFragment(savedInstanceState == null, false) {
            HistoryChart()
        }
        setupToolbar()
    }

    override val drawToBottomEdge = false
}